src/screens/ChannelListScreen/index.tsx

```diff
@@ -94,6 +95,7 @@ const ChannelListScreen = ({route}) => {
           />
         }
         renderItem={renderChannelItem}
+        style={{backgroundColor: COLORS.almostBlack}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `style` prop of the component has been updated to set the background color to `COLORS.almostBlack`. Ensure that this color change aligns with your app's overall theme and doesn't affect readability or user experience.
```diff
+ style={{backgroundColor: COLORS.almostBlack}}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->